### PR TITLE
Bump rails 6.1.4

### PR DESCRIPTION
**Pull request by hannako**
## What

Apply rails app:update changes

## Why

When [this](https://github.com/alphagov/link-checker-api/pull/434) dependabot PR was merged, the gemfile was updated to 6.1, but the config defaults for 6.1 were not updated. This PR updates them as per the [Rails guidance](https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#moving-between-versions).

- Trello: https://trello.com/c/nWRFaLO3/2562-manually-bump-apps-to-rails-61

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -21,15 +24,24 @@ module LinkCheckerApi
   self.hosts_with_basic_authorization = {}
 
   class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
-    config.load_defaults = 6.0
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.0
```
Comment (hannako) [inline config/application.rb]: Weirdly `config.load_defaults` was set to 5.0 when running app:update after updating the gemfile to 6.1 🤷‍♀️ 

I manually changed it to 6.0 and ran the tests, but they continued to fail

Comment (ChrisBAshton) [reply config/application.rb]: I've come across this before too 🤔 Not sure why!

File: config/cable.yml [hunk 2]
```diff
@@ -2,8 +2,9 @@ development:
...
 
 test:
-  adapter: async
+  adapter: test
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: link_checker_api_production
```
Comment (ChrisBAshton) [inline config/cable.yml:10]: Was this change automatic? Quite surprised to see the application name in the default output!

**Review by ChrisBAshton — APPROVED** [review]
LGTM 👍 nice one!

File: config/cable.yml (hunk 2 above)
Comment (hannako) [reply config/cable.yml:10]: Yes that was automatic

**Pull request merged**